test/fixtures/RetrievePensionChargesFixtures.scala

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
can you move this to test/data rather than test/fixtures please? Also delete the fixture folder.